index.html

```diff
@@ -637,13 +637,15 @@ <h3>Proofs</h3>
           <dt><dfn class="lint-ignore">created</dfn></dt>
           <dd>
 The date and time the proof was created is OPTIONAL and, if included, MUST be
-specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string.
+specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string and SHOULD include a
+zone offset, preferably Z (meaning UTC).
```

Comment:
```suggestion
specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string, either in Universal Coordinated Time (UTC), denoted by a Z at the end of the value, or with a time zone offset relative to UTC. Time values that are incorrectly serialized without an offset MUST be interpreted as UTC.
```

Reply:
@msporny's suggestion with line breaks as recommended...
```suggestion
specified as an [[XMLSCHEMA11-2]] `dateTimeStamp` string, either in Universal
Coordinated Time (UTC), denoted by a Z at the end of the value, or with a time
zone offset relative to UTC. Time values that are incorrectly serialized without
an offset MUST be interpreted as UTC.
```